inc/commonglpi.class.php

```diff
@@ -690,6 +690,10 @@ function showTabsContent($options = []) {
          if ($this->getType() == 'Ticket') {
             $this->input = $cleaned_options;
             $this->saveInput();
+            if (isset($cleaned_options['is_locked'])) {
+               $extraparamhtml = "&amp;".Toolbox::append_params([$cleaned_options['is_locked']],
```

Comment:
the option must be like ['locked' => 1] and not [$cleaned_options['is_locked']], otherwise the extraparam is '0=1' when it should be 'locked=1'